spec/deskpro_spec.rb

```diff
@@ -4,22 +4,24 @@
 
 	describe "Ticket.new" do
 
-		it "should create a ticket with read accessors" do
-			ticket = Deskpro::Ticket.new({
-				person_email: "test@localhost.local",
-				subject: "HELP",
-				message: "Please help me",
-				agent_team_id: 1,
-				message_as_agent: 1,
-				label: ['problem', 'production'],
-			})
-			expect(ticket.person_email).to eq('test@localhost.local')
-			expect(ticket.subject).to eq('HELP')
-			expect(ticket.message).to eq('Please help me')
-			expect(ticket.agent_team_id).to eq(1)
-			expect(ticket.message_as_agent).to eq(1)
-			expect(ticket.label).to eq(['problem', 'production'])
-		end
+        it "should create a ticket with read accessors" do
```

Comment:
We've got a mixture of tabs and spaces—new content has spaces, existing content is tabbed.

Reply:
I've noticed this in the HTML content too.